integrations/extensions/starter-kits/watson-discovery/README.md

```diff
@@ -41,9 +41,13 @@ More broadly, anything you can do with Watson Discovery parameters and Watson Di
+Before starting, you will need an instance of [IBM Watson Discovery](https://www.ibm.com/cloud/watson-discovery).  You will also need the URL for that instance and either an API key or a bearer token for that instance.  Typically API keys are used for IBM Cloud instances and bearer tokens are used for instances running in Cloud Pak for Data outside of the IBM Cloud.  Both the URL and the API key or bearer token are found on the credentials page for the instance on IBM Cloud or on Cloud Pak for Data.
 
-You will also need to load some data into IBM Watson Discovery.  The example actions in this starter kit were tested on an IBM Watson Discovery project in which we ingested the page `https://bitcoin.org/en/faq` and configured Discovery to follow 0 links (i.e., load that one page only) with FAQ Extraction enabled to split that page into multiple question/answer pairs.  For more details on FAQ Extraction in Watson Discovery, see [Turn your FAQ pages into conversational AI](https://medium.com/ibm-watson/turn-your-faq-pages-into-conversational-ai-8ac7ae7ec793).  To try out this starter kit, you can also ingest that example page, or you can connect directly to your own data and modify the details of the starter kit as needed to be relevant to your data.
+You will also need to load some data into IBM Watson Discovery.  The example actions in this starter kit were tested on an IBM Watson Discovery project in which we ingested the page `https://bitcoin.org/en/faq` and configured Discovery to follow 0 links (i.e., load that one page only).  To try out this starter kit, you can also ingest that example page, or you can connect directly to your own data and modify the details of the starter kit as needed to be relevant to your data.  Either way, be sure to get the project ID for the project containing your data.
+
+If you are using Watson Discovery in Cloud Pak for Data, this kit will only work if your Watson Discovery instance has a valid SSL certificate signed by a trusted authority _or_ you are connecting to Watson Discovery using a Watson Assistant on Cloud Pak for Data version 4.6.3 or later.  In practice, we find that it is rare for anyone to have a Cloud Pak for Data instance that has a valid SSL certificate signed by a trusted authority, so this using this kit with Cloud Pak for Data _mostly_ only works for version 4.6.3 or later.  This issue does not apply if you are running Watson Discovery in the public cloud, because the public cloud has a valid certificate signed by a trusted authority.
```

Comment:
```suggestion
If you are using Watson Discovery in Cloud Pak for Data, this kit will only work if your Watson Discovery instance has a valid SSL certificate signed by a trusted authority _or_ you are connecting to Watson Discovery using a Watson Assistant on Cloud Pak for Data version 4.6.3 or later.  In practice, we find that it is rare for anyone to have a Cloud Pak for Data instance that has a valid SSL certificate signed by a trusted authority, so using this kit with Cloud Pak for Data _mostly_ only works for version 4.6.3 or later.  This issue does not apply if you are running Watson Discovery in the public cloud, because the public cloud has a valid certificate signed by a trusted authority.
```